plugin-build/plugin/src/main/kotlin/com/mikepenz/aboutlibraries/plugin/mapping/SpdxLicense.kt

```diff
@@ -239,7 +246,14 @@ enum class SpdxLicense(
     mpich2("mpich2 License", "mpich2"),
     MPL_1_0("Mozilla Public License 1.0", "MPL-1.0"),
     MPL_1_1("Mozilla Public License 1.1", "MPL-1.1"),
-    MPL_2_0("Mozilla Public License 2.0", "MPL-2.0"),
+    MPL_2_0("Mozilla Public License 2.0", "MPL-2.0",
+        customMatcher = { name, _ ->
```

Comment:
I think matching for the url to contains `mozilla.org/MPL/2.0` is probably an easier and more secure approach in that case. (also should be cheaper than constructing a regex)

Reply:
Thanks. 

if only all would use spdx identifiers :D